vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/Popover.java

```diff
@@ -70,10 +73,80 @@ public class Popover extends Component implements HasAriaLabel, HasComponents {
     public Popover() {
         getElement().getNode().addAttachListener(this::attachComponentRenderer);
 
+        // Workaround for: https://github.com/vaadin/flow/issues/3496
+        getElement().setProperty("opened", false);
+
         updateTrigger();
         setOverlayRole("dialog");
     }
 
+
+    /**
+     * {@code opened-changed} event is sent when the overlay opened state
+     * changes.
+     */
+    public static class OpenedChangeEvent extends ComponentEvent<Popover> {
```

Comment:
This needs to be annotated with `@DomEvent("opened-changed")` now, otherwise the event listener won't work.